src/components/hardware-wallets/NanoXDeviceAnimation.tsx

```diff
@@ -21,7 +21,12 @@ import gridDotsLight from '@/assets/dot-grid-light.png';
 import gridDotsDark from '@/assets/dot-grid-dark.png';
 import { ImgixImage } from '@/components/images';
 import { useColorMode } from '@/design-system';
-import { useDimensions } from '@/hooks';
+import { deviceUtils } from '@/utils';
+
+export const scaleFactor = deviceUtils.isSmallPhone ? 0.9 : 1;
+
+export const ledgerNanoHeight = 292 * scaleFactor;
+export const ledgerNanoWidth = 216 * scaleFactor;
```

Comment:
Might be cool to make those capitalized like `SCALE_FACTOR` will be easy to spot constants in the code right away. This is a nit so decide yourself

Reply:
yes